src/systems/renderer.js

```diff
@@ -1,6 +1,7 @@
 var registerSystem = require('../core/system').registerSystem;
 var utils = require('../utils/');
 var THREE = require('../lib/three');
+var {aframeSortTransparentDefault, aframeSortTransparentSpatial} = require('../core/scene/sortFunctions');
```

Comment:
I believe destructuring is an ES6 feature.

Reply:
Calling `require` more than once on the same module gives the same result, but I think it's a tad cleaner to assign it to an intermediate, i.e. `var sortFunctions = require('../core/scene/sortFunctions')`